test/Scripts.Integration.Test/Editor/Builder.cs

```diff
@@ -16,21 +16,21 @@ public static void BuildIl2CPPPlayer(BuildTarget target, BuildTargetGroup group)
         // Make sure the configuration is right.
         EditorUserBuildSettings.selectedBuildTargetGroup = group;
         PlayerSettings.SetScriptingBackend(group, ScriptingImplementation.IL2CPP);
-
-        if (args.ContainsKey("sentryOptions.configure"))
-        {
-            SetupSentryOptions(args);
-        }
+        DisableUnityAudio();
 
         var buildPlayerOptions = new BuildPlayerOptions
         {
-            scenes = new[] { "Assets/Scenes/1_Bugfarm.unity" },
             locationPathName = args["buildPath"],
             target = target,
             targetGroup = group,
             options = BuildOptions.StrictMode,
         };
 
+        if(File.Exists("Assets/Scenes/SmokeTest.unity"))
+        {
+            buildPlayerOptions.scenes = new[] { "Assets/Scenes/SmokeTest.unity" };
+        }
```

Comment:
It's not there when we build the clean project on main.

Reply:
We use the `Builder.cs` to build a project without Sentry. In that case, we have not yet copied the SmokeTest-related scripts/scenes and the build would fail.
https://github.com/getsentry/sentry-unity/blob/1fb22e5f6b1517d3e9f8eef4758bb6d1cd207c0e/.github/workflows/ci.yml#L240

Reply:
Is there actually anything related to smoke tester in that scene? AFAIK it's only the SmokeTester.cs that's needed. Can we thus have the scene there all the time? I assume the game scene doesn't influence that? 

My POV of what the integration test with those two builds should do is test that if we add Sentry to an existing game, then it works fine and doesn't break. In that case it makes sense for the project only to change as necessary between the two builds.

Reply:
The way it's set up right now is that the `SmokeTest.scene` has a GameObject with the `SmokeTester.cs` component. And the SmokeTester has Sentry as a dependency, which is why it gets added during the Sentry SDK setup and why it gets added only after we install the package.

But this is a good point and will be a great optimization and cut down on Unity runs (which cuts down on time. A lot). I'll put that into its own issue.